README.md

```diff
@@ -342,6 +341,40 @@ extension MyViewController: ShopifyCheckoutSheetKitDelegate {
 }
 ```
 
+### Error handling
+
+In the event of a checkout error ocurring, the Checkout Sheet Kit _may_ attempt a retry to recover from the error. Recovery will happen in the background by discarding the failed webview and creating a new "recovery" instance. Recovery will be attempted in the following scenarios:
```

Comment:
apparently it's occurring and not ocurring

Reply:
Ah, I need a spellcheck in vscode 😄

Reply:
Updated